src/Orckestra.Composer.Search/Providers/FromPriceProvider.cs

```diff
@@ -17,10 +20,14 @@ public sealed class FromPriceProvider : IPriceProvider
 
         //TODO: To be refactored. Composer Context should NEVER be referenced from here.
         private IComposerContext ComposerContext { get; }
```

Comment:
Please remove ComposerContext property, as it isn't used. Along with the constructor parameter and the TODO comment